prototype_source/quantization_2_0_tutotial.rst

```diff
@@ -0,0 +1,567 @@
+which uses symbolic_trace to capture the model into a graph, and then
+perform quantization transformations on top of the captured model. In a
+similar way, for Quantization 2.0 flow, we will now use the PT2 Export
+workflow to capture the model into a graph, and perform quantizations
+transformations on top of the ATen dialect graph. This is expected to
+have significantly higher model coverage, better programmability, and
+a simplified UX.
+
+Suppose we are a backend developer and we wish to integrate our backend
+with PyTorch's quantization 2.0 flow. We only need to define the backend
+specific quantizer. An existing quantizer object defined for
+QNNPack/XNNPack is here
+`QNNPackQuantizer <https://github.com/pytorch/pytorch/blob/main/torch/ao/quantization/_pt2e/quantizer/qnnpack_quantizer.py>`__.
+Taking QNNPackQuantizer as example, the overall Quantization 2.0 flow could be:
+
+::
+
+    import torch
+    import torch._dynamo as torchdynamo
+    from torch.ao.quantization._quantize_pt2e import convert_pt2e, prepare_pt2e
+    import torch.ao.quantization._pt2e.quantizer.qnnpack_quantizer as qq
+
+    class M(torch.nn.Module):
+        def __init__(self):
+            super().__init__()
+            self.linear = torch.nn.Linear(5, 10)
+
+        def forward(self, x):
+            return self.linear(x)
+
+    example_inputs = (torch.randn(1, 5),)
+    model = M().eval()
+
+    # Step 1: Trace the model into an FX graph of flattened ATen operators
+    exported_graph_module, guards = torchdynamo.export(
+        model,
+        *copy.deepcopy(example_inputs),
+        aten_graph=True,
+    )
+
+    # Step 2: Insert observers or fake quantize modules
+    quantizer = qq.QNNPackQuantizer()
+    operator_config = qq.get_symmetric_quantization_config(is_per_channel=True)
+    quantizer.set_global(operator_config)
+    prepared_graph_module = prepare_pt2e(exported_graph_module, quantizer)
+
+    # Step 3: Quantize the model
+    convered_graph_module = convert_pt2e(prepared_graph_module)
+
+    # Step 4: Lower Reference Quantized Model into the backend
+
+Inside the Quantizer, we will use the `QuantizationAnnotation API <https://docs.google.com/document/d/1tjIsL7-uVgm_1bv_kUK7iovP6G1D5zcbzwEcmYEG2Js/edit#>`__
+to convey user's intent for what quantization spec to use and how to
+observe certain tensor values in the prepare step. Now, we will have a step by step
+tutorial for how to use the `QuantizationAnnotation API` to create a quantizer.
+
+1. Define QuantizationConfig
+--------------------------------------------------------
+
+QuantizationConfig defines the data type and qscheme for activation, weight and bias. Suppose
+we want to define:
+
+-  Activation: `int8` data type, `per_tensor_affine` quantization, `HistogramObserver`
+-  Weight    : `int8` data type, `per_channel_symmetric` quantization, `PerChannelMinMaxObserver`
+-  Bias      : `float` data type, `PlaceholderObserver`
```

Comment:
Can we explain how the qconfig defined here how the defined `QuantizationSpec` here are used or related to the `QuantizationSpec`  used for annotating the model.

Reply:
Add more explanation for how `QuantizationConfig` is defined and how does `QuantizationSpec` been used to create observer when annotating the model.